test/utils/describeConformanceV5.js

```diff
@@ -192,6 +193,38 @@ function testThemeComponents(element, getOptions) {
   });
 }
 
+/**
+ * Tests that the root component has the root class
+ * @param {React.ReactElement} element
+ * @param {() => ConformanceOptions} getOptions
+ */
+export function testRootClass(element, getOptions) {
```

Comment:
Did you consider host the logic inside describeConformance.js? I wonder because it seems to lead to fewer changes, and that `describeConformanceV5.js` is a temporary helper, until we migrate all the components.

Reply:
Moved the test in `describeConformance` 👍 The first time I tried I got lot of failiars, but seems like it was because of the `Ripple`.